src/v3/src/components/Select/Select.tsx

```diff
@@ -78,11 +79,11 @@ const Select: UISchemaElementComponent<UISchemaElementComponentWithValidationPro
         );
         handleChange?.(selectTarget.value);
       }}
-      onBlur={(e: SelectChangeEvent<string>) => {
+      onBlur={(e: FocusEvent) => {
         const selectTarget = (
           e?.target as SelectChangeEvent['target'] & { value: string; name: string; }
         );
-        handleBlur?.(selectTarget.value);
+        handleBlur?.(selectTarget.value, e);
```

Comment:
I think this should have been `e.currentTarget` (which is guaranteed to be on `e` instead of `e.target`, which is nullable), but perhaps @trevoring-okta can comment on if using `e.target` was intentional.